src/anyvar/utils/liftover_utils.py

```diff
@@ -0,0 +1,244 @@
+
+    error_details = "Could not convert refget accession"
+
+
+def _convert_coordinate(converter: Converter, chromosome: str, coordinate: int) -> int:
+    """Convert an individual coordinate to another reference genome. If the conversion is unsuccessful, raises a `CoordinateConversionError`
+
+    :param converter: An AGCT Converter instance.
+    :param chromosome: The chromosome number where the coordinate is found. Must be a string consisting of a) the prefix "chr", and b) a number OR "X" or "Y" -> i.e. "chr10".
+    :param coordinate: A single start or end coordinate. MUST be an `int`.
+
+    :return: A converted coordinate value as an `int`
+    :raises: A `CoordinateConversionError` if the conversion is unsuccessful.
+    """
+    converted_position = converter.convert_coordinate(
+        chromosome, coordinate, Strand.POSITIVE
+    )
+
+    # TODO: Handle cases where coordinate conversion returns negative-stranded coordinates. See Issue #197.
+    if converted_position and converted_position[0][2] == Strand.POSITIVE:
+        # TODO: Don't just return coordinates from the first result set - handle cases where coordinate map to multiple positions. See Issue #198.
+        return converted_position[0][1]
+    raise CoordinateConversionError
+
+
+_PositionType = TypeVar("_PositionType", int, list[int | None])
+
+
+def convert_position(
+    converter: Converter, chromosome: str, position: _PositionType
+) -> _PositionType:
+    """Convert a SequenceLocation position (i.e., `start` or `end`) to another reference Genome. `position` can either be a `list[int | None]` or an `int` - return type will match.
+
+    :param converter: An AGCT Converter instance.
+    :param chromosome: The chromosome number where the position is found. Must be a string consisting of a) the prefix "chr", and b) a number OR "X" or "Y" -> e.g. "chr10", "chrX", etc.
+    :param position: A SequenceLocation start or end position. Can be a `list[int | None]` or an `int`.
+
+    :return: A lifted-over position. Type (`list[int | None]` or `int`) will match that of `position`
+    """
+    # Handle int positions
+    if isinstance(position, int):
+        return _convert_coordinate(converter, chromosome, position)
+
+    # Handle Range (list) positions
+    lower_bound, upper_bound = position
+    lower_bound = (
+        _convert_coordinate(converter, chromosome, lower_bound) if lower_bound else None
+    )
+    upper_bound = (
+        _convert_coordinate(converter, chromosome, upper_bound) if upper_bound else None
+    )
+    return [lower_bound, upper_bound]
+
+
+def get_liftover_variant(variant_object: dict, anyvar: AnyVar) -> VrsVariation:
+    """Liftover a variant from GRCh37 or GRCH38 into the opposite assembly, and return the converted variant as a VrsObject.
+    If liftover is unsuccessful, raise an Exception.
+
+    :param variant_object: A dictionary representation of a `VrsVariation`.
+    :param seqrepo_dataproxy: A `SeqrepoDataproxy` instance.
```

Comment:
It's Nitpicky to comment on a comment, but seqrepo_dataproxy is coming from AnyVar not a param